Composer/packages/server/src/models/bot/builder.ts

```diff
@@ -121,7 +121,16 @@ export class Builder {
       await this.runQnaBuild(interruptionQnaFiles);
       await this.runOrchestratorBuild(orchestratorBuildFiles, emptyFiles);
     } catch (error) {
-      throw new Error(error.message ?? error.text ?? 'Error publishing to LUIS or QNA.');
+      // handle this special error case where QnA Maker returns this uninformative error.
+      // in their portal, it is accompanied by a message about the search service limits
+      // (A free search is limited to 3 indexes, which can be used up quickly)
+      if (error.text === 'Qnamaker build failed: Runtime error.') {
```

Comment:
is the error always going to be in English?

Reply:
The error from Qnamaker?

Or are you saying we should formatMessage this error?

Reply:
former.

I see that the runQnaBuild is sending a ```culture``` that is basically a locale

Reply:
That is a good question! I don't know how to change that setting...